happybase/thrift_sasl.py

```diff
@@ -0,0 +1,169 @@
+""" SASL transports for Thrift. """
```

Comment:
wow. what a bunch of custom code. is this based on thrift code since it seems to use its (internal?) classes and exceptions? if so, attribution please. :)